framework/src/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -133,6 +134,9 @@ public ValueAnnotatedTypeFactory(BaseTypeChecker checker) {
         // this line just registers the alias. The BottomVal is never used.
         addAliasedAnnotation(MinLen.class, BOTTOMVAL);
 
+        addAliasedAnnotation(
+                "org.checkerframework.checker.index.qual.Positive", createIntRangeFromPositive());
```

Comment:
Thanks for the clarification.  Could you add a comment in the code explaining that?